agent/authenticator/authenticator_suite_test.go

```diff
@@ -1,112 +0,0 @@
-// Copyright 2022 VMware, Inc. All Rights Reserved.
```

Comment:
All the `agent/authenticator/*` are removed because the Admission controller has required functionality, did I get that right?

Reply:
The same is added now in `agent/main.go` and `agent/registration/csr.go`